dali-mqtt-daemon.py

```diff
@@ -253,9 +257,19 @@ def create_mqtt_client(
     parser.add_argument(
         "--ha-discover-prefix", help="HA discover mqtt prefix", default="homeassistant"
     )
+    parser.add_argument("--log-color", help="Coloring output", action="store_true")
```

Comment:
store_false (that should be the default)

Reply:
You are sure? `store_true` pass `True` to `args.log_color` when option `--log-color` is passed (and output is colored)
So colored output is optional and disabled by default (as described in #16?)

Reply:
sorry, misread "action"/"default"